include/deal.II/multigrid/mg_transfer.h

```diff
@@ -143,6 +147,53 @@ namespace internal
   };
 
 #endif
+
+#ifdef DEAL_II_WITH_PETSC
+  template <>
+  struct MatrixSelector<dealii::PETScWrappers::MPI::Vector>
+  {
+    typedef ::dealii::DynamicSparsityPattern Sparsity;
+    typedef ::dealii::PETScWrappers::MPI::SparseMatrix Matrix;
+
+    // Since PETSc matrices do not offer the functionality to fill up incomplete sparsity patterns on their own, the sparsity pattern must be manually distributed
+    // This only works if the row index set is known, which can be extracted from a DynamicSparsityPattern but not the other types
+    // The support of MatrixSelectore is therefore limited to that type for PETSc
```

Comment:
would you please mind breaking the comment lines at 80 characters?

Reply:
Done. Can I simply push this into the feature branch or will that cause unnecessary load on the test infrastructure?

Reply:
Yes sure, please go ahead. There is usually enough resources - if not, testing will simply take some more hours - and we do this all the time.